pkg/resource.go

```diff
@@ -259,9 +259,11 @@ func (res *URLResource) Load() ([]byte, error) {
 // NewGITResourceBundle will create a new instance of GITResourceBundle
 // url is the GIT http/https url.
 // pathPattern are list of file pattern (glob) to filter files located in the repository
-func NewGITResourceBundle(url string, pathPattern ...string) *GITResourceBundle {
+func NewGITResourceBundle(url string, user string, password string, pathPattern ...string) *GITResourceBundle {
```

Comment:
Please change the function name to other name and keep the original function as it is. To avoid function break.